### PR TITLE
Add the "$" to the commands to documentation and update talk links

### PR DESCRIPTION
**Pull request by gkapfham**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -387,6 +387,16 @@ listed in alphabetical order by their surname.
 - Race Mahoney
 - Nicholas Tocci
 
+## Presentations
+
+GatorGrader's creators give presentations about the development, use, and
+assessment of the tool. Please contact one of the developers if you would like
+to feature a presentation about GatorGrader at your technical conference. The
+following list includes some recent of our presentations:
```
Comment (Michionlion) [inline README.md]: Probably should be reworded, perhaps to "The following list includes some of our recent presentations"

File: README.md [hunk 2]
```diff
@@ -245,7 +245,7 @@ line in your terminal window, we give them on multiple lines to ensure that
 they are easily viewed on a wide variety of displays.
 
 ```
-pipenv run python3 gatorgrader.py \
+$ pipenv run python3 gatorgrader.py \
```
Comment (Michionlion) [inline README.md]: This looks good. Placing them on each line would make copy-pasting inconvenient, so I think this strategy should work well!

File: README.md (hunk 1 above)
Comment (gkapfham) [reply README.md]: Thanks for pointing this out. I fixed it!

File: README.md (hunk 2 above)
Comment (gkapfham) [reply README.md]: Great, thanks for the confirmation.

**Pull request merged**